### DISCUSSION TITLE
Facebook Inbox Message API

### DISCUSSION THREAD

**Discussion by matheusggds** [frontendbr/forum#821] [Discussão]
Boa noite comunidade!

Então, vocês são minha última esperança, por favor se puder me ajudar eu vou ser eternamente grato, preciso de verdade dessa help, valeu!

Eu possuo um aplicativo registrado no Facebook, para poder fazer a integração com meu sistema web. Eu necessito que quando o usuário fizer login no meu sistema utilizando sua conta no Facebook, eu consiga filtrar seus amigos por quais deles já possuem sua conta no face vinculada com o meu app(do face). Até ai não tive problemas, consigo essa lista tranquilamente através desta [api](https://developers.facebook.com/docs/graph-api/reference/user/friends/), mas aqui vai o bixo de 7 cabeças.

Eu preciso conseguir mandar uma mensagem inbox para essas pessoas (sim, eu possuo o ID do Facebook delas) com uma mensagem personalizada. Eu achei está [API](https://developers.facebook.com/docs/sharing/reference/send-dialog) no Facebook que funcionaria perfeitamente, o único problema é que no atribute "to" eu não consigo passar mais de um ID de usuário, como se eu tivesse selecionado as pessoas para qual quero mandar a mensagem ='(

Alguém já teve que fazer alguma aplicação que necessitasse deste tipo de integração? Eu não sei mais o que fazer, e preciso muuuuito, é para um cliente muito grande que não posso revelar o nome. 

Mesmo que não possa me ajudar, MUITO obrigado por ter lido tudo, valeu a intenção. 

Abraço pra todos galera!

**Comment by fernandomoraes** [comment]
Oi, já trabalhei um tempo com essa api, talvez consiga te dar um help.

A parte que você menciona :
> eu não consigo passar mais de um ID de usuário, como se eu tivesse selecionado as pessoas para qual quero mandar a mensagem ='(

Para somente um usuário, você consegue? Te questiono isso, pois pelo oque lembro, o id que tu usa pra enviar mensagem não é o mesmo id do facebook do usuário, isso evitaria de tu conseguir enviar mensagens através de um bot, para pessoas que nunca conversou contigo, fazendo tipo um spam.
